pkgs/by-name/nf/nfs-ganesha/package.nix

```
@@ -16,6 +16,7 @@
   flex,
   nfs-utils,
   acl,
+  ceph,
```

```suggestion
  useCeph ? false,
  ceph,
```
Let's make it conditional. This is a backend that pulls in a lot of dependencies.